CONTRIBUTING.md

```diff
@@ -14,23 +12,32 @@ Make sure the tests pass:
 
     rake
 
-Make your change. Add tests for your change. Make the tests pass:
+Make your change.
+Write tests.
+Follow our [style guide][style].
+Make the tests pass:
+
+[style]: https://github.com/thoughtbot/guides/tree/master/style
 
     rake
 
-Push to your fork and [submit a pull request][pr].
+Write a [good commit message][commit].
+Push to your fork.
+[Submit a pull request][pr].
 
+[commit]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
 [pr]: https://github.com/thoughtbot/suspenders/compare/
 
-At this point you're waiting on us. We like to at least comment on pull requests
-within three business days (and, typically, one business day). We may suggest
-some changes or improvements or alternatives.
+If [Hound] catches style violations,
+fix them.
 
-Some things that will increase the chance that your pull request is accepted:
+[hound]: https://houndci.com
 
-* Write tests.
-* Follow our [style guide][style].
-* Write a [good commit message][commit].
+Wait for us.
+We try to at least comment on within one business day.
```

Comment:
I feel like this sentence is missing a word somewhere. We try to at least comment on *new pull requests* within one business day?